### PR TITLE
Adapt example notebooks to release

### PR DESCRIPTION
**Pull request by ferrigno**
Need to verify my modification, as the archive is not mounted.

### CODE REVIEW

**Comment by ferrigno** [comment]
I have reviewed the notebooks and they seem to all work.
I have also changed a default value to display images.

**Comment by ferrigno** [comment]
@volodymyrss I am waiting for your review

File: doc/source/user_guide/AsynchronousAPI.ipynb [hunk 1]
```diff
@@ -201,1629 +218,973 @@
...
-    "            \n",
-    "            if '-' not in osa_version:\n",
-    "                if year >= 2016:\n",
-    "                    osa_version='OSA11.0'\n",
-    "                else:\n",
-    "                    osa_version='OSA10.2'\n",
-    "                    \n",
-    "            ys = (year, semester)\n",
+    "        T1_utc='%4d-01-01T00:00:00.0'%year\n",
+    "        T2_utc='%4d-12-31T23:59:59.0'%year # there are 30 days in June!\n",
```
Comment (volodymyrss) [inline doc/source/user_guide/AsynchronousAPI.ipynb:1147]: there are, however, 31 days in December

File: doc/source/user_guide/AsynchronousAPI.ipynb [hunk 2]
```diff
@@ -201,1629 +218,973 @@
...
-      "048300640010\n",
-      "048300650010\n",
-      "048300660010\n",
-      "048300670010\n",
-      "048300680010\n",
-      "Length = 34 rows\n",
-      "Is complete True\n",
-      "3 4 5\n",
-      "complete 6 / 6\n",
+      "Is complete  True\n"
```
Comment (volodymyrss) [inline doc/source/user_guide/AsynchronousAPI.ipynb:947]: it's pretty clear that this loop over multiple request args is very reusable and could be all in a function like `get_product_set()`. But users might want to implement some custom async requests too, so it's not bad to have this custom example. I would even keep this example and then add at the end "by the way, we made a convenience function for this". For the future.

File: doc/source/user_guide/Authentication.ipynb [hunk 3]
```diff
@@ -21,11 +24,11 @@
    "source": [
     "The first step is obtaining a token.\n",
     "First, you need to \"Sign In\" on the frontend. \n",
-    "[https://frontend.staging.odahub.io/cdci/astrooda/](https://frontend.staging.odahub.io/cdci/astrooda/)\n",
+    "[https://www.astro.unige.ch/mmoda/](https://www.astro.unige.ch/mmoda/)\n",
     "\n",
     "### Create account\n",
     "\n",
-    "If you do not have an account create one, by clicking always on Sign-in.\n",
+    "If you do not have an account create one, by clicking always on Sign-in nad then create new account.\n",
```
Comment (volodymyrss) [inline doc/source/user_guide/Authentication.ipynb:31]: `nad` => `and`

**Review by volodymyrss — CHANGES_REQUESTED** [review]
some comments to Async apply also to Auth.
Thanks!

File: doc/source/user_guide/AsynchronousAPI.ipynb [hunk 4]
```diff
@@ -45,73 +54,16 @@
     }
    ],
    "source": [
-    "#You can provide a valid token as explained in the 'Authentication' example or skip this cell\n",
     "import getpass\n",
     "token = getpass.getpass('Insert the token')"
```
[the remark this answers is not in the document]
Comment (ferrigno) [reply doc/source/user_guide/AsynchronousAPI.ipynb:58]: What is 'exp' ? in decode token?

discover_token() returns None , what is it about?

oda_api.token.decode_oda_token(token)

{'sub': 'Carlo.Ferrigno@unige.ch',
 'email': 'Carlo.Ferrigno@unige.ch',
 'name': 'cferrigno',
 'roles': 'authenticated user, administrator, content manager, general, integral-private-qla, magic, unige-hpc-full, public-pool-hpc, antares, sdss',
 'exp': 1631110932}

[the remark this answers is not in the document]
Comment (burnout87) [reply doc/source/user_guide/AsynchronousAPI.ipynb:58]: It's the expiration time in seconds.

[the remark this answers is not in the document]
Comment (volodymyrss) [reply doc/source/user_guide/AsynchronousAPI.ipynb:58]: > What is 'exp' ? in decode token?

that's right, time when the token expires, in unix seconds. Can be used as so:

https://github.com/oda-hub/oda_api/blob/5c762edd597d7bbc8573c94d03df906ba1b10891/oda_api/cli.py#L86-L91
(I guess let's not document cli yet)

> discover_token() returns None , what is it about?

it means there is no token in the system.

you can see the methods of discovery in here:

https://github.com/oda-hub/oda_api/blob/5c762edd597d7bbc8573c94d03df906ba1b10891/oda_api/token.py#L75-L83

it should be relatively clear what are the 3 methods:

* ODA_TOKEN env variable
* ./.oda-token file
* ~/.oda-token file


> 
> oda_api.token.decode_oda_token(token)
> 
> {'sub': '[Carlo.Ferrigno@unige.ch](mailto:Carlo.Ferrigno@unige.ch)',
> 'email': '[Carlo.Ferrigno@unige.ch](mailto:Carlo.Ferrigno@unige.ch)',
> 'name': 'cferrigno',
> 'roles': 'authenticated user, administrator, content manager, general, integral-private-qla, magic, unige-hpc-full, public-pool-hpc, antares, sdss',
> 'exp': 1631110932}

**Pull request merged**